test/metric_value_benchmark/statsd_test.go

```diff
@@ -44,62 +70,122 @@ func (t *StatsdTestRunner) GetAgentConfigFileName() string {
 }
 
 func (t *StatsdTestRunner) GetAgentRunDuration() time.Duration {
-	return time.Minute
+	return 3 * time.Minute
```

Comment:
Wanted a few minutes to verify multiple data points and the data points are every 30 seconds. I don't want to lower that since I want some aggregation to occur as well.